moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1349,6 +1349,9 @@ bool RobotState::getJacobian(const JointModelGroup* group, const LinkModel* link
                              const Eigen::Vector3d& reference_point_position, Eigen::MatrixXd& jacobian,
                              bool use_quaternion_representation) const
 {
+  using LinkModel = moveit::core::LinkModel;
```

Comment:
Is this necessary? I think this already is in the moveit core namespace?

Reply:
ah, good catch!
removed the unnecessary namespace qualification here and in other places in this file.